site/docs/downloads/index.md

```diff
@@ -22,118 +22,234 @@ See also the [Nessie release page page for version {{ versions.nessie }} on GitH
 
 The main Nessie server serves the Nessie repository using the Iceberg REST API and Nessie's native REST API.
 
-### Docker image
+=== "Docker Image"
 
-Docker images are multiplatform images for amd64, arm64, ppc64le, s390x.
+    Docker images are multiplatform images for amd64, arm64, ppc64le and s390x.
+    They are available from the following repositories:
+    
+    * [GitHub Container Registry](https://ghcr.io/projectnessie/nessie):
 
-=== "GitHub Container Registry"
-    [GitHub Container Registry](https://ghcr.io/projectnessie/nessie)
     ```bash
     docker pull ghcr.io/projectnessie/nessie:{{ versions.nessie }}
     docker run -p 19120:19120 -p 9000:9000 ghcr.io/projectnessie/nessie:{{ versions.nessie }}
     ```
-=== "Quay.io"
-    [quay.io](https://quay.io/repository/projectnessie/nessie?tab=tags)
+    
+    * [Quay.io](https://quay.io/repository/projectnessie/nessie?tab=tags):
+    
     ```bash
     docker pull quay.io/projectnessie/nessie:{{ versions.nessie }}
     docker run -p 19120:19120 -p 9000:9000 quay.io/projectnessie/nessie:{{ versions.nessie }}
     ```
 
-### Helm Chart
+=== "Helm Chart"
 
-=== "Artifact Hub"
-    [Artifact Hub](https://artifacthub.io/packages/search?repo=nessie)
-=== "Nessie Helmchart Repo"
-    [https://charts.projectnessie.org/](https://charts.projectnessie.org/)
-=== "Tarball"
-    [Nessie {{ versions.nessie }} Helm Chart](https://github.com/projectnessie/nessie/releases/download/nessie-{{ versions.nessie }}/nessie-helm-{{ versions.nessie }}.tgz)
+    Nessie {{ versions.nessie }} Helm chart is available from the following locations:
 
-### Standalone uber jar
+    * [Nessie Helm Repo](https://charts.projectnessie.org/):
+    
+    ```bash
+    helm repo add nessie https://charts.projectnessie.org/
+    helm repo update
+    helm install my-nessie nessie/nessie --version "{{ versions.nessie }}"
+    ```
+
+    * [Artifact Hub](https://artifacthub.io/packages/helm/nessie/nessie).
+    * [Nessie {{ versions.nessie }} Helm Chart Tarball](https://github.com/projectnessie/nessie/releases/download/nessie-{{ versions.nessie }}/nessie-helm-{{ versions.nessie }}.tgz).
+
+=== "Standalone Jar"
+
+    Requires Java 17 or newer.
+    
+    ```bash
+    curl -L -o nessie-quarkus-{{ versions.nessie }}-runner.jar \
+      https://github.com/projectnessie/nessie/releases/download/nessie-{{ versions.nessie }}/nessie-quarkus-{{ versions.nessie }}-runner.jar
+    java -jar nessie-quarkus-{{ versions.nessie }}-runner.jar
+    ```
+
+=== "Maven Central"
```

Comment:
Not sure whether Maven coordinates for a distributable are helpful. I suspect it would rather confuse people. Better remove those from here and elsewhere.

The Snapshot-Repo information on the in-dev/index.md is fine. Should be made clearer though that this is meant for developing against Nessie - but maybe also include the `nessie-bom` as an `enforcedPlatform()` for Gradle and `<dependencyManagement>` for Maven poms. Might also be helpful to add a similar section to the release downloads.

Reply:
> Not sure whether Maven coordinates for a distributable are helpful. I suspect it would rather confuse people.

If someone needs to build a custom Nessie server, e.g. to add more extensions, that's exactly the snippet they would need to include, no?

Reply:
Anyways I removed the Maven Central tabs.

I also tried to implement all your suggestions (not sure I got all of them right.)